server/data/localMockData/identifiersMock.ts

```diff
@@ -0,0 +1,59 @@
+import { OffenderIdentifier } from '../../interfaces/prisonApi/offenderIdentifier'
+
+// eslint-disable-next-line import/prefer-default-export
+export const identifiersMock: OffenderIdentifier[] = [
+  {
+    type: 'CRO',
+    value: '400862/08W',
+    offenderNo: 'G6123VU',
+    issuedAuthorityText: 'P/CONS',
+    caseloadType: 'INST',
+    whenCreated: '2012-02-13T13:48:31.650303',
+  },
+  {
+    type: 'PNC',
+    value: '08/359381C',
+    offenderNo: 'G6123VU',
+    issuedAuthorityText: 'P/CONS',
+    caseloadType: 'INST',
+    whenCreated: '2012-02-13T13:48:31.664201',
+  },
+  {
+    type: 'MERGED',
+    value: 'A0596CL',
+    offenderNo: 'G6123VU',
+    caseloadType: 'INST',
+    whenCreated: '2012-02-13T13:48:43.632903',
+  },
+  {
+    type: 'MERGE_HMPS',
+    value: 'JK7713',
+    offenderNo: 'G6123VU',
+    caseloadType: 'INST',
+    whenCreated: '2012-02-13T13:48:43.636262',
+  },
+  {
+    type: 'DL',
+    value: 'ABCD/123456/AB9DE',
+    offenderNo: 'G6123VU',
+    issuedDate: '2016-09-08',
+    caseloadType: 'INST',
+    whenCreated: '2016-09-08T09:15:32.160802',
+  },
+  {
+    type: 'NINO',
+    value: 'AB123456A',
```

Comment:
Is it worth using the fake one here? - `QQ123456C`

not sure how important it is that we dont use a real one 🤷🏻 but that's the fake one, AB123456A might actually belong to someone (I know AB123456C does)

Reply:
Just copied and pasted from other mock data and assumed it was fake. Best to change to a known fake one - updating now.